### DISCUSSION TITLE
How does docusaurus sync with the source code?

### DISCUSSION THREAD

**Discussion by iamnewton** [facebook/docusaurus#3722] [General] [answered]
I’m not sure that its the intention of Docusaurus, but is there a way to maintain any sort of synchronization with the source code that is intended to be documented?

**Comment by slorber** [comment, marked as answer]
It's not clear to me what you mean by synchronization.

If the code and the doc site live in the same git repo,  they remain in sync automatically. 

If it's not the case, then you have the responsibility to maintain sync and I don't have any specific solution to suggest, as this problem is not really related to Docusaurus


If you want to show a code block of an actual code file, here's an example: https://v2.docusaurus.io/examples/markdownpageexample/#import-code-block-from-source-code-file
